public/app/features/alerting/unified/api/alertSilencesApi.ts

```diff
@@ -0,0 +1,65 @@
+import { Silence, SilenceCreatePayload } from 'app/plugins/datasource/alertmanager/types';
+
+import { alertingApi } from './alertingApi';
+
+export const alertSilencesApi = alertingApi.injectEndpoints({
+  endpoints: (build) => ({
+    getSilences: build.query<
+      Silence[],
+      {
+        datasourceUid: string;
+      }
+    >({
+      query: ({ datasourceUid }) => ({
+        url: `/api/alertmanager/${datasourceUid}/api/v2/silences`,
+      }),
+      providesTags: ['AlertmanagerSilences'],
+    }),
+
+    getSilence: build.query<
+      Silence,
+      {
+        datasourceUid: string;
+        id: string;
+      }
+    >({
+      query: ({ datasourceUid, id }) => ({
+        url: `/api/alertmanager/${datasourceUid}/api/v2/silence/${id}`,
+      }),
+      providesTags: ['AlertmanagerSilences'],
```

Comment:
I can add this in :+1: but for now we would still need to invalidate the "list" of silences as well so the UI refetches the table view properly. But it makes sense to start being more comprehensive about our cache tags here

Reply:
Done - let me know your thoughts

Reply:
I've been going back and forth on this since revising the tags in the settings PR and I understand now that it was actually unnecessary for it to be separate tags but it doesn't hurt much either :D implementation LGTM!